src/describe.cpp

```diff
@@ -1,6 +1,27 @@
 #include <Omega_h_element.hpp>
 #include <Omega_h_file.hpp>
+#include <Omega_h_tag.hpp>
+#include <Omega_h_array_ops.hpp>
 
+template <typename T>
+void printTagInfo(Omega_h::Mesh mesh, std::ostringstream& oss, int dim, int tag, std::string type) {
+    auto tagbase = mesh.get_tag(dim, tag);
+    auto array = Omega_h::as<T>(tagbase)->array();
+
+    Omega_h::Real min = get_min(array);
+    Omega_h::Real max = get_max(array);
+
+    oss << "(" << tagbase->name().c_str() << ", " << dim << ", " << type << ")\n";
+    oss << "\tNum Components: " << tagbase->ncomps() << "\n";
+    oss << "\tMin, Max: " << min << ", " << max << "\n\n";
```

Comment:
I think we should combine these values onto one line (with a modified header) to make the output more compact and more likely to fit on one 'screen'.